docs/test_conversation_history.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
Looks identical to the notebook? Let's keep only one copy of the logic?

Reply:
The notebook didn't work. Was picking agentai library version and not the folder. So, had to use the python file to test things out

Reply:
` pip install -e .` might be worth looking into for local development